modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -183,85 +202,90 @@
 /datum/job/cyborg
 	alt_titles = list(
 		"Cyborg",
-		"Robot",
 		"Android",
+		"Machine",
```

Comment:
This is an odd choice. Calling a somewhat "sentient" piece of moving, talking robot a mere "machine". That just larps it with every other machine on the station. Not really a fan of this.

Reply:
Plenty of synth players, myself included, play synths that cant be called sentient or sapient, or would perfer to at least pretend its the case

Reply:
I think that Machine is better when I can use it as an insult and it's not actually just the truth :trollface: